src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -145,8 +145,8 @@
     <value>Usage: pwsh[.exe] [-Login] [[-File] &lt;filePath&gt; [args]]
                   [-Command { - | &lt;script-block&gt; [-args &lt;arg-array&gt;]
                                 | &lt;string&gt; [&lt;CommandParameters&gt;] } ]
-                  [-ConfigurationName &lt;string&gt;] [-CustomPipeName &lt;string&gt;]
-                  [-EncodedCommand &lt;Base64EncodedCommand&gt;]
+                  [-ConfigurationName &lt;string&gt;] [-ConfigurationFile &lt;filePath&gt;]
```

Comment:
Typo? I'd expect ABC order.

Reply:
We are not consistent on the ABC order elsewhere. I think it is fine as is.